addon/utils/e3-easing.js

```diff
@@ -0,0 +1,16 @@
+const EASING = {
+  'ease-in-cubic': function(percent) {
```

Comment:
I think there actually is a difference between `ease-in-cubic` and plan old `cubic` (as there should also be a corresponding `ease-out-cubic`). 

http://gizma.com/easing/#cub3 shows the differences.